Lib/fontTools/feaLib/ast.py

```diff
@@ -421,6 +462,9 @@ def addDefinition(self, definition):
             self.glyphs[glyph] = definition
 
     def glyphSet(self):
+        """The set of glyphs contained in this mark class. In the example above,
```

Comment:
It’s a set in the general sense but a tuple in the Python sense. Or maybe that is an implementation detail. What kind of collection does this method, in principle, return?

Reply:
It needs to be ordered. I'll bring this into line with the other classes which have `glyphSet` methods.